docs/mobile-apps/automated-testing/ipa-files.md

```diff
@@ -8,38 +8,31 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This topic covers:
-* For Appium: How to create an iOS .ipa file using self-signed application files.
-* For XCUITest: How to create iOS .app files with our XCUITest Real Device Test Runner and Virtual Device Test Runner.
-
 We use a resigning process to embed our own provisioning profile into the uploaded .ipa file. With your own profile in place, the app will be allowed to install and launch on cloud devices. The functionality of the app itself remains completely unmodified.
```

Comment:
I know you didn't edit this paragraph, but is `your` a typo in the second sentence? It doesn't make sense to me --  _"We use...our own provisioning profile..." With **your** own profile in place...."_

Are we saying that the ipa need 2 profiles? Ours AND yours?

Reply:
I changed "your own" to "the." I think that should solve/sidestep the issue so I can get this PR merged.